teuthology/task/install.py

```diff
@@ -927,6 +926,121 @@ def _upgrade_deb_packages(ctx, config, remote, debs):
     )
 
 
+@contextlib.contextmanager
+def rh_install(ctx, config):
+    """
+    Installs rh ceph on all hosts in ctx.
+
+    :param ctx: the argparse.Namespace object
+    :param config: the config dict
+    """
+    version = config['rhbuild']
+    rh_versions = ['1.3.0']
+    if version in rh_versions:
+        log.info("Supported version")
+    else:
+        raise RuntimeError("Unsupported RH Ceph version %s " , version)
+
+    with parallel() as p:
+        for remote in ctx.cluster.remotes.iterkeys():
+            if remote.os.name == 'rhel':
+                log.info("Installing on RHEL node")
+                p.spawn(rh_install_pkgs, ctx, remote, version)
+            else:
+                raise RuntimeError("Test requires RHEL nodes")
```

Comment:
I think the log does indicate the last error message and its easier to know anyway